hydromt_delft3dfm/dflowfm.py

```diff
@@ -1487,6 +1458,385 @@ def setup_1dboundary(
             da_out, name=f"boundary1d_{da_out.name}_{branch_type}"
         )  # FIXME: this format cannot be read back due to lack of branch type info from model files
 
+    def _read_forcing_geodataset(
+        self, forcing_geodataset_fn: Union[str, Path], forcing_name: str = "discharge"
+    ):
+        """Read forcing geodataset."""
+
+        refdate, tstart, tstop = self.get_model_time()  # time slice
+
+        if (
+            forcing_geodataset_fn is not None
+            and self.data_catalog[forcing_geodataset_fn].data_type == "GeoDataset"
+        ):
+            da = self.data_catalog.get_geodataset(
+                forcing_geodataset_fn,
+                geom=self.region.buffer(
+                    1000
+                ),  # only select data within region of interest (large region for forcing)
+                variables=[forcing_name],
+                time_tuple=(tstart, tstop),
+            ).rename(forcing_name)
```

Comment:
Do you need the rename here? Or rather should be done in the data catalog instead?
Normally you read data with hydromt standard variable name --> process data in workflow --> rename to your model names and had to model components like forcing

Reply:
Check, I think here the renaming is not needed.